app/controllers/conference_controller.rb

```diff
@@ -15,6 +15,9 @@ def schedule
     @rooms = @conference.venue.rooms if @conference.venue
     @events = @conference.program.events
     @dates = @conference.start_date..@conference.end_date
+    confirmed_events = @conference.program.events.where(state: 'confirmed')
+    @scheduled_events = confirmed_events.where('start_time IS NOT NULL').order(:start_time)
```

Comment:
We already have btw the following scopes:
https://github.com/openSUSE/osem/blob/master/app/models/program.rb#L29
https://github.com/openSUSE/osem/blob/master/app/models/program.rb#L25